src/pcapi/admin/custom_views/category_view.py

```diff
@@ -36,6 +38,8 @@ def subcategories(self) -> Response:
         return self.render(
             "admin/subcategories_list.html",
             subcategories=ALL_SUBCATEGORIES,
+            searchGroups=SearchGroups,
+            HomepageLabels=HomepageLabels,
```

Comment:
```suggestion
            search_groups=SearchGroups,
            homepage_labels=HomepageLabels,
```
Ce sont des variables Python passées au contexte.